README.md

```diff
@@ -207,25 +207,25 @@ Sponsors provide support for `fgbio` through direct funding or employing contrib
 Public sponsors include:
 
 <p>
-<a href float="left"="https://fulcrumgenomics.com"><img src=".github/logos/fulcrumgenomics.svg" alt="Fulcrum Genomics" height="35"/></a>
+<a href="https://fulcrumgenomics.com"><img src=".github/logos/fulcrumgenomics.svg" alt="Fulcrum Genomics" height="35"/></a>
 &nbsp;
-<a href float="left"="https://twinstrandbio.com/"><img src=".github/logos/twinstrandbio.svg" alt="TwinStrand Biosciences" height="45"/></a>
+<a href="https://twinstrandbio.com/"><img src=".github/logos/twinstrandbio.svg" alt="TwinStrand Biosciences" height="45"/></a>
 &nbsp;
-<a href float="left"="https://www.jumpcodegenomics.com//"><img src=".github/logos/jumpcodegenomics.png" alt="Jumpcode Genomics" height="30"/></a>
+<a href="https://www.jumpcodegenomics.com//"><img src=".github/logos/jumpcodegenomics.png" alt="Jumpcode Genomics" height="30"/></a>
 &nbsp;
-<a href float="left"="https://www.igenomx.com//"><img src=".github/logos/igenomx.png" alt="iGenomX" height="30"/></a>
+<a href="https://investors.twistbioscience.com/news-releases/news-release-details/twist-bioscience-acquires-ngs-library-preparation-maker-igenomx"><img src=".github/logos/igenomx.png" alt="iGenomX" height="30"/></a>
```

Comment:
I wasn't sure what link to use since the original link does not exist anymore.

I figured this press release was the next best thing.